packages/calling/src/Voicemail/Voicemail.ts

```diff
@@ -115,12 +123,12 @@ export class Voicemail extends Eventing<VoicemailEventTypes> implements IVoicema
   }
 
   /**
-   * Call voicemail class to fetch the voicemail lists.
+   * Retrieves a list of voicemails with optional pagination and sorting options.
    *
-   * @param sort - Sort voicemail list (ASC | DESC). TODO: Once we start implementing sorting.
    * @param offset - Number of records to skip.
-   * @param offsetLimit - Number of voicemail list to fetch from the offset.
-   * @param refresh - Refresh the list of voicemails from backend.
+   * @param offsetLimit - The limit on the number of voicemails to retrieve from the offset.
+   * @param sort - Sort voicemail list (ASC | DESC). TODO: Once we start implementing sorting.
```

Comment:
TODO ?

Reply:
It was there in the old doc. Will check and update this. @Kesari3008 do you know if this is a still a TODO?

Reply:
@sreenara  Should be able to comment on it. I am not sure why we had this TODO in place. But even if we are keeping the TODO it should not be present as part of these descriptions. Should be present as a separate commented line which shouldn't appear in API doc